src/game.cxx

```diff
@@ -206,7 +207,8 @@ void Game::run()
     evManager.checkEvents(event, engine);
 
     // render the tileMap
-    engine.map->renderMap();
+    if (engine.map != nullptr)
```

Comment:
it's enough to check
`if (engine.map)`

you don't need to check against nullptr

> 4.12 Boolean conversions
> 1 A prvalue of arithmetic, unscoped enumeration, pointer, or pointer to member type can be converted to a prvalue of type bool. A zero value, null pointer value, or null member pointer value is converted to false; any other value is converted to true. For direct-initialization (8.5), a prvalue of type std::nullptr_t can be converted to a prvalue of type bool; the resulting value is false.

Reply:
yeah I know that 
that way it's more understandable to newbies or juniors, this is the reason I did that.